npm-shrinkwrap.json

```diff
@@ -4626,7 +4625,7 @@
         "cross-spawn": "^5.1.0",
         "debug": "^3.1.0",
         "doctrine": "^2.1.0",
-        "eslint-scope": "3.7.1",
+        "eslint-scope": "^3.7.1",
```

Comment:
Do we still need to fix the version here?

Reply:
_need_ to, not really. but we probably should.